pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.wavefront</groupId>
+    <artifactId>dropwizard-metrics</artifactId>
```

Comment:
There is already a `dropwizard-metrics` library published under `com.wavefront` currently at version 4.30: http://mvnrepository.com/artifact/com.wavefront/dropwizard-metrics

Should be okay continuing that if we bump up version to > 4.30, but we should rename if we want this to be a brand new artifact.

Reply:
I will call the new one "dropwizard-metrics-sdk"